docs/source/api/api.rst

```diff
@@ -560,4 +560,143 @@ Example: GET /locations/
           "name": "Newport"
         }
       ]
-    }
\ No newline at end of file
+    }
+
+Themes listing
+----------------
+
+The themes listing is available at ``/themes/``. It returns a JSON array
+consisting of each of the themes, and information about them.
+
```

Comment:
I would add information about how colors are returned -- that they're ``rgb(x, y, z)``. (What's the name for this string encoding that distinguishes it from ``#ABCDEF``?)